docs/typescript/how-to-define-workflow-parameters-in-typescript.md

```diff
@@ -0,0 +1,35 @@
+---
+id: how-to-define-workflow-parameters-in-typescript
+title: How to define workflow parameters in Typescript
+sidebar_label: Define Workflow parameters
+description: Define Workflow parameters
+tags:
+  - developer-guide
+  - sdk
+  - typescript
+---
+
+A Task Queue is a dynamic queue in Temporal polled by one or more Workers.
```

Comment:
How to set a Task Queue is an h4 section of How to start a Workflow Execution - 
Defining Workflow parameters is about making sure you are using serializable objects so that you can change the fields in the object without breaking the workflow function signature down the road.

Its just a solid best practice that we want to be in the habit of showing - too many samples show the usage of strings as the workflow params.